### DISCUSSION TITLE
Just asking a question. Im a nubee. Lol easy on my stupity.;)

### DISCUSSION THREAD

**Discussion by Buffalo-716** [termux/termux-packages#10917] [Q&A] [answered]
So how would i use termux to access and operate  /system/priv-app/EngineerMode on my zte z3153v using android 10 not rooted but has fakeroot installed in termux. Thanks for your help im worried if i root phone i might brick it. ( I dont have access to a computer only my phone here...). Can it be done even with chroot?

**Comment by ChiefMikeK** [comment, marked as answer]
Just wondering why you want to use this app that root exploits your phone via adb?

Interesting read:  
[OnePlus Device Backdoor Root Exploits EngineerMode App | NowSecure](https://www.nowsecure.com/blog/2017/11/14/oneplus-device-root-exploit-backdoor-engineermode-app-diagnostics-mode/ )

**Reply by Buffalo-716** [reply]
Well to be honest with you I'm new at this I've tried to access engineering mode with code and it will not allow me if literally tried everything. I've written ZTE and they are no help . My phone has features that I want access to ie  CDMA volume adjustment antenna adjustment for better signal. I have bought my phone so technically it is mine so I should be able to do as I wish with it. So I'm looking for the best way to do what I need to do with having limited access to things. My biggest issue is not having access to a computer  so I figured this program should be able to help me do what I need to be done . That is why I chose this program. If you know a better way to do things and safer way  I'm open to it.

**Reply by ChiefMikeK** [reply]
AS PER my understanding that program needs a PC to access your phone to root exploit it and then the PC can access those settings.  
Sorry but I don't have a solution 4u but can try searching/asking 4 advice on a few forums and possibly some YouTube Tech/cellphone repair channels(in-comments).
- https://www.reddit.com/search?q=zte+android&restrict_sr=&sort=relevance&t=all
- https://www.xda-developers.com/tag/zte/
- https://m.youtube.com/results?sp=mAEA&search_query=android+repair

**Reply by Buffalo-716** [reply]
Can it be done using another phone and wifi it to the other phone kinda like an ota? Or what about using an SD card to side load it? I've been reading alot and am learning more and more daily
